src/unit_tests/wazuh_modules/vulnerability_detector/test_wm_vuln_detector.c

```diff
@@ -23180,6 +23210,7 @@ int main(void)
         cmocka_unit_test(test_wm_vuldet_set_feed_update_url_redhat_greater_5),
         cmocka_unit_test(test_wm_vuldet_set_feed_update_url_suse),
         cmocka_unit_test(test_wm_vuldet_set_feed_update_url_suse_invalid),
+        cmocka_unit_test(test_wm_vuldet_set_feed_update_url_alma),
```

Comment:
A unit test is missing to cover the case that the URL is from AlmaLinux but with an invalid version: `test_wm_vuldet_set_feed_update_url_alma_invalid`